Common/Util/LeanData.cs

```diff
@@ -1013,7 +1013,12 @@ public static bool TryParseSecurityType(string fileName, out SecurityType securi
                 var fileName = Path.GetFileNameWithoutExtension(filePath);
                 if (fileName.Contains("_"))
                 {
-                    tickType = (TickType)Enum.Parse(typeof(TickType), fileName.Split('_')[1], true);
+                    var tickTypePosition = 1;
```

Comment:
😔, hate that this is so niche and specific. Would be good to have examples documented of each type of filename format.

Reply:
yeah, it's all centralized logic now in `LeanData`, covered by the new unit tests too 👍